docs/core/integration-quality-scale/rules/iqs012.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "IQS012 - Entities use has_entity_name = True"
+---
+
+IQS012 is as follows:
+
+Entities use has_entity_name = True
+
+## Reasoning
+
+`has_entity_name` is an entity attribute, which enables a new naming method.
+It is introduced to split the name of the entity from the name of the device or service.
+With `has_entity_name` set to `True`, the user will have a better experience on the device page, since not every entity is prefixed with the device name.
+This feature also allowed users to rename a device and automatically update all entities that belong to that device.
```

Comment:
This also isn't true since that feature existed before this attribute was introduced.